apis/v1alpha1/instrumentation_types.go

```diff
@@ -22,6 +22,10 @@ import (
 
 // InstrumentationSpec defines the desired state of OpenTelemetry SDK and instrumentation.
 type InstrumentationSpec struct {
+	// Selector is the selector label of injected Object
+	// +optional
```

Comment:
```suggestion
	// Selector is the label selector for affected Pods.
	// Unlike standard label selectors, `nil` means `everything`, and this is also the default.
	// This may change in a future CRD version.
	// +optional
```